hpack.cabal

```diff
@@ -33,7 +33,7 @@ library
     , containers
     , unordered-containers
     , yaml
-    , aeson >= 0.11
+    , aeson >= 0.8
   exposed-modules:
```

Comment:
this was an issue with a previous patch, can you remove this change from your PR